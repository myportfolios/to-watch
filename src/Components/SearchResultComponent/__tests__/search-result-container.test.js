import React from "react";
import { shallow } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import {
  SearchResultContainer,
  mapStateToProps
} from "../SearchResultContainer";

configure({ adapter: new Adapter() });

describe("Search Result Container", () => {
  const props = {
    searchResult: [
      {
        title: "Wake up"
      }
    ]
  };
  let wrapper = shallow(<SearchResultContainer {...props} />);

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
  xit("should mapStateToProps correctly", () => {
    const state = {
      searchMovie: {
        data: [
          {
            title: "wake up"
          }
        ]
      }
    };
    expect(mapStateToProps(state)).toEqual({
      searchResult: state.searchMovie.data
    });
  });
});
