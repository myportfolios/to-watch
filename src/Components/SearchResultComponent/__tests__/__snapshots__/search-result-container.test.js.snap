// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Result Container renders correctly 1`] = `ShallowWrapper {}`;
